org.scala-ide.sdt.core/src/org/scalaide/refactoring/internal/OrganizeImports.scala

```diff
@@ -212,7 +215,7 @@ class OrganizeImports extends RefactoringExecutorWithoutWizard {
     }
   }
 
-  class OrganizeImportsScalaIdeRefactoring(file: ScalaSourceFile) extends ScalaIdeRefactoring("Organize Imports", file, 0, 0) {
+  class OrganizeImportsScalaIdeRefactoring(file: ScalaSourceFile) extends ScalaIdeRefactoring(OrganizeImports, "Organize Imports", file, 0, 0) {
```

Comment:
it's not the only place in code but what for you need `"Organize Imports"` if you have `OrganizeImports` enum? above is `MoveConstructorToCompanion` same case and a few more I guess